core/controllers/admin.py

```diff
@@ -729,6 +730,56 @@ def post(self):
         self.render_json({})
 
 
+class AdminGrantSuperAdminPrivilegesHandler(base.BaseHandler):
+    """Handler for granting a user super admin privileges."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        if self.email != feconf.ADMIN_EMAIL_ADDRESS:
+            raise self.UnauthorizedUserException(
+                'Only the default system admin can manage super admins')
+
+        username = self.request.get('username', None)
+        if username is None:
+            raise self.InvalidInputException('Missing username param')
+
+        user_id = user_services.get_user_id_from_username(username)
+        if user_id is None:
+            raise self.InvalidInputException('No such user exists')
+
+        auth_services.grant_super_admin_privileges(user_id)
+        self.render_json(self.values)
+
+
+class AdminRevokeSuperAdminPrivilegesHandler(base.BaseHandler):
```

Comment:
Why not have one handler (SuperAdminRoleHandler) with `put` and `delete` methods?

Reply:
Great idea, done!